input/blog/2018-08-21-cake-v0.30.0-released.md

```diff
@@ -0,0 +1,76 @@
+﻿---
+title: Cake v0.30.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.30.0 of Cake has been released.
+
+With this release we've got several improvements and bug fixes contributed from the community, so firstly big _**thanks**_ to all for your continued support!
+
+In this post we would like to highlight that Cake is now available as a global .NET CLI tool.
+
+This means you can easily install Cake and have it globally available in your environment using the `dotnet tool` command.
+
+```
+dotnet tool install -g Cake.Tool --version 0.30.0
+```
+
+Once installed, then you can launch `cake` from anywhere using the .NET CLI
+```
+dotnet cake build.cake
+```
+
+Which means you can now easily bootstrap the installation of Cake, if you have .NET Core SDK version 2.1.400 or newer installed.
+Existing boostrappers will still work as we created a new NuGet package for the tool, which means we now have the Cake packages as
+
+* [Cake](https://www.nuget.org/packages/Cake) - compiled for .NET 4.6.1 / Mono
+* [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR) - compiled for .NET Core 2.0
+* [Cake.Tool](https://www.nuget.org/packages/Cakle.Tool) - .NET tool compiled for .NET Core 2.1
+
+Contributions were included from:
+
+- [devlead](https://github.com/devlead)
+- [Marusyk](https://github.com/Marusyk)
+- [gep13](https://github.com/gep13)
+- [SIkebe](https://github.com/SIkebe)
+- [mholo65](https://github.com/mholo65)
+- [jnm2](https://github.com/jnm2)
+- [AdaskoTheBeAsT](https://github.com/AdaskoTheBeAsT)
+- [patriksvensson](https://github.com/patriksvensson)
+- [gitfool](https://github.com/gitfool)
+- [pascalberger](https://github.com/pascalberger)
+
+Full details of everything that was included in this release can be seen below.
+
+<!--excerpt-->
+
+### Issues
+
+As part of this release we had [14 issues](https://github.com/cake-build/cake/milestone/53?closed=1) closed.
+
+__Feature__
+
+- [__#2067__](https://github.com/cake-build/cake/issues/2067) Publish as .NET Core Global Tool.
+
+__Improvement__
+
+- [__#2238__](https://github.com/cake-build/cake/issues/2238) Add repository metadata to NuGet packages.
+- [__#2234__](https://github.com/cake-build/cake/issues/2234) Remove mono argument from Argument Parser.
```

Comment:
What happends if the argument is still passed? Does it throw an exception? In this case it might be better to classify this as a breaking change since it might require changes on the user side

Reply:
No it won't throw anything. Will be treated as any argument.